src/sidebar/components/user-menu.js

```diff
@@ -50,6 +51,11 @@ function UserMenu({ auth, bridge, onLogout, serviceUrl, settings }) {
     !isThirdParty || serviceSupports('onProfileRequestProvided');
   const isLogoutEnabled =
     !isThirdParty || serviceSupports('onLogoutRequestProvided');
+  const isNotebookEnabled = features.flagEnabled('notebook_launch');
```

Comment:
Use `useStore(store => store.isFeatureEnabled(...))` instead. That ensures that the component is re-rendered if the value changes.

Reply:
Argh :). We should add some comments to the `features` service to this effect—I'm pretty sure this has bitten me before.